pages/mesosphere/dcos/2.1/security/ent/tls-ssl/ar-custom/index.md

```diff
@@ -88,6 +88,7 @@ client. The server names can include:
   - `master.mesos`
   - `leader.mesos`
   - `registry.component.thisdcos.directory`
+  - any IP address returned by the `/opt/mesosphere/bin/detect_ip` script (created from the `ip_detect_contents` or `ip_detect_filename` configuration parameter).
```

Comment:
There is always one such adress I think
```suggestion
  - an IP address returned by the `/opt/mesosphere/bin/detect_ip` script (created from the `ip_detect_contents` or `ip_detect_filename` configuration parameter).
```

Reply:
There's a different address returned on each master - the certificate should not contain any of these. I'll let @cmastr decide if this is clearer either way.

Reply:
Any makes more sense to me.